Dshop/apps/products_catalogue/models.py

```diff
@@ -117,20 +131,24 @@ def get_attributes(self):
         return attributes
 
     def get_price(self, item: CartItem) -> DecimalField:
-
         return self.price
 
     def is_available(self):
         return self.availability
 
 
 class ProductImage(models.Model):
```

Comment:
Everything seems fine. Wondering if there is a way to test files for Products (and if it should be done)

Reply:
Thank you for review. What do you mean by testing iles for Products? Please elaborate.